account_journal_lock_date/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_update_journal_lock_dates_wizard,access_update_journal_lock_dates_wizard,model_update_journal_lock_dates_wizard,base.group_user,1,1,1,1
```

Comment:
In the wizard there is a check on being a member of account.group_account_manager. Would it perhaps make more sense to apply that group here as well? Otherwise the action might be visible to users who cannot execute it.

Reply:
@StefanRijnhart Group changed in permissions by group account.group_account_manager

Reply:
Perfect, thanks!